src/main/kotlin/com/woowacourse/archive/ConversationService.kt

```diff
@@ -31,35 +29,42 @@ class ConversationService(
     }
 
     private fun getLastConversationTime(savedConversations: List<Conversation>) =
-            savedConversations[savedConversations.size - 1].conversationTime.toString()
+        savedConversations[savedConversations.size - 1].conversationTime.toString()
 
-    private fun saveAll(conversations: Conversations): MutableList<Conversation> {
-        return repository.saveAll(toList(conversations))
-    }
+    private fun saveAll(conversations: Conversations): MutableList<Conversation> =
+        repository.saveAll(toList(conversations))
 
     private fun toList(conversations: Conversations): List<Conversation> {
         return conversations
-                .conversations
-                .stream()
-                .map { to(it) }
-                .collect(toList())
+            .conversations.stream()
```

Comment:
Kotlin의 `Sequence`를 사용하면 어떨까요?

```suggestion
            .conversations.asSequence()
```

Reply:
https://proandroiddev.com/java-streams-vs-kotlin-sequences-c9ae080abfdc

오호 그렇군요. 감사합니다

Reply:
두 군데 반영하여 수정했습니다.